arguebuf/models/graph.py

```diff
@@ -708,6 +708,43 @@ def from_ova(
 
         return g
 
+    @classmethod
+    def from_sadface(
+            cls,
+            obj: sadface.Graph,
+            name: t.Optional[str] = None,
+            atom_class: t.Type[AtomNode] = AtomNode,
+            scheme_class: t.Type[SchemeNode] = SchemeNode,
+            edge_class: t.Type[Edge] = Edge,
+            nlp: t.Optional[t.Callable[[str], t.Any]] = None,
+    ) -> Graph:
+        """Generate Graph structure from SADFace argument graph file
+        (reference: https://github.com/Open-Argumentation/SADFace/blob/master/examples/hangback/data.json).
+        """
+        g = cls(name)
```

Comment:
Metadata is missing here. [Created](https://github.com/Open-Argumentation/SADFace/blob/c17fd89e3f6355cbe9bc4bd320bb1461894e2752/examples/hangback/data.json#L33) and [updated](https://github.com/Open-Argumentation/SADFace/blob/c17fd89e3f6355cbe9bc4bd320bb1461894e2752/examples/hangback/data.json#L35) timestamps are available